### PR TITLE
Add VCS support to meld

### PR DESCRIPTION
**Pull request by rusty-snake**

### CODE REVIEW

File: etc/meld.profile [hunk 1]
```diff
@@ -10,7 +10,7 @@ noblacklist ${HOME}/.local/share/meld
 noblacklist ${PATH}/python*
 noblacklist /usr/include/python*
 noblacklist /usr/lib/python*
-noblacklist /usr/local/lib/python*
+noblacklist /usr/local/lib/python*,git,svn,hg,bzr,cvs
```
Comment (SkewedZeppelin) [inline etc/meld.profile]: find all and replace?

Comment (rusty-snake) [reply etc/meld.profile]: not entered, but maybe a button accidentally pressed (using vim). will be removed.

**Comment by glitsj16** [comment]
Personally I find the meld profile a bit too restrictive for a diff tool. Nothing that can't be handled via a local override, but I find myself diff'ing files blacklisted by `disable-common.inc` and `disable-programs.inc` quite often. Just a thought...

**Comment by rusty-snake** [comment]
I would add it with a little more feedback.

**Comment by rusty-snake** [comment]
I only tested git, because I don't use svn,hg,bzr,cvs / have no experience.

**Comment by rusty-snake** [comment]
From my site it can be merged.

**Comment by glitsj16** [comment]
@rusty-snake For consistency's sake, would you mind using the same format as other profiles do that need to allow python? I mean with reagards to commenting and spacing. Also would be nice to alphabetically order private-bin and private-etc. Here's an [example](https://git.io/fjTkt
).

**Comment by rusty-snake** [comment]
@glitsj16 done, thanks for the preliminary work.

**Comment by glitsj16** [comment]
Merged, thanks!

**Pull request merged**